### ISSUE TITLE
Employee can not see the profile

### ISSUE THREAD

**Issue by don-lin** [don-lin/group10-healingpaws#2]
If an employee enter the profile page, the authorization will be cancelled.

**Comment by Kory777** [comment]
Yes, that's because we only have one profile page in routes, and we can solve it by adding a "employe profile" or inserting some codes in "profile" to keep authorization.
